src/main/java/ru/yandex/practicum/filmorate/storage/film/InMemoryFilmStorage.java

```diff
@@ -1,32 +1,27 @@
-package ru.yandex.practicum.filmorate.controller;
+package ru.yandex.practicum.filmorate.storage.film;
 
-import jakarta.validation.Valid;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.stereotype.Component;
 import ru.yandex.practicum.filmorate.exception.NotFoundException;
 import ru.yandex.practicum.filmorate.model.Film;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
 @Slf4j
-@RestController
-@RequestMapping("/films")
-public class FilmController {
-    private final Map<Integer, Film> films = new HashMap<>();
+@Component
+public class InMemoryFilmStorage implements FilmStorage {
+    private final Map<Long, Film> films = new HashMap<>();
 
-    @PostMapping
-    public Film create(@Valid @RequestBody Film film) {
+    @Override
+    public Film create(Film film) {
         Film createdFilm = film.toBuilder().id(getNextId()).build();
```

Comment:
Лучше разить билдер по строкам, это повысит читаемость кода:
```suggestion
        Film createdFilm = film.toBuilder()
            .id(getNextId())
            .build();
```